Surgery Merger

mazinskihenry opened the pull request:
Starting Work

mazinskihenry committed:
Starting

mazinskihenry committed:
Most stuff is here

MiszczuZPolski commented on addons/surgery/ACE_Medical_Treatment_Actions.hpp:
```diff
@@ -0,0 +1,207 @@
+    class CPR;
+
+    class BreathCheck: Carbonate {
+        displayName = "Check Breath";
```
CSTRING

MiszczuZPolski commented on addons/surgery/ACE_Medical_Treatment_Actions.hpp:
```diff
@@ -0,0 +1,207 @@
+        animationMedicProne = "";
+    };
+    class Dialysis: Defibrillator_AED_X_vehicle {
+        displayName = "Perform Dialysis";
```
CSTRING

MiszczuZPolski commented on addons/surgery/ACE_Medical_Treatment_Actions.hpp:
```diff
@@ -0,0 +1,207 @@
+    };
+    class Dialysis: Defibrillator_AED_X_vehicle {
+        displayName = "Perform Dialysis";
+        displayNameProgress = "Performing";
```
CSTRING

MiszczuZPolski commented on addons/surgery/ACE_Medical_Treatment_Actions.hpp:
```diff
@@ -0,0 +1,207 @@
+        icon = QPATHTOF(ui\X_Series-Device_W.paa);
+    };
+    class CheckFracture: CheckPulse {
+        displayName = "Check Fracture";
```
CSTRING

MiszczuZPolski commented on addons/surgery/ACE_Medical_Treatment_Actions.hpp:
```diff
@@ -0,0 +1,207 @@
+    };
+    class CheckFracture: CheckPulse {
+        displayName = "Check Fracture";
+        displayNameProgress = "Checking Fracture";
```
CSTRING

MiszczuZPolski commented on addons/surgery/ACE_Medical_Treatment_Actions.hpp:
```diff
@@ -0,0 +1,207 @@
+        callbackSuccess = QUOTE([ARR_3(_medic, _patient, _bodyPart)] call FUNC(fractureSelect));
+    };
+    class ClosedReduction: CheckFracture {
+        displayName = "Perform Closed Reduction";
```
CSTRING

MiszczuZPolski commented on addons/surgery/ACE_Medical_Treatment_Actions.hpp:
```diff
@@ -0,0 +1,207 @@
+    };
+    class ClosedReduction: CheckFracture {
+        displayName = "Perform Closed Reduction";
+        displayNameProgress = "Performing Reduction";
```
CSTRING

MiszczuZPolski commented on addons/surgery/ACE_Medical_Treatment_Actions.hpp:
```diff
@@ -0,0 +1,207 @@
+        callbackSuccess = QUOTE([ARR_3(_medic, _patient, _bodyPart)] call FUNC(closedFracture));
+    };
+    class OpenReduction: CheckFracture {
+        displayName = "Perform Open Reduction";
```
CSTRING

MiszczuZPolski commented on addons/surgery/ACE_Medical_Treatment_Actions.hpp:
```diff
@@ -0,0 +1,207 @@
+    };
+    class OpenReduction: CheckFracture {
+        displayName = "Perform Open Reduction";
+        displayNameProgress = "Performing Reduction";
```
CSTRING

MiszczuZPolski commented on addons/surgery/ACE_Medical_Treatment_Actions.hpp:
```diff
@@ -0,0 +1,207 @@
+        callbackSuccess = QUOTE([ARR_3(_medic, _patient, _bodyPart)] call FUNC(openFracture));
+    };
+    class Etomidate: Carbonate {
+        displayName = "Push Etomidate";
```
CSTRING

MiszczuZPolski commented on addons/surgery/ACE_Medical_Treatment_Actions.hpp:
```diff
@@ -0,0 +1,207 @@
+        callbackSuccess = "[_medic, _patient, _bodyPart, _className, _itemUser, _usedItem] call ace_medical_treatment_fnc_medication;";
+    };
+    class Lorazepam: Carbonate {
+        displayName = "Push Lorazepam";
```
CSTRING

MiszczuZPolski commented on addons/surgery/ACE_Medical_Treatment_Actions.hpp:
```diff
@@ -0,0 +1,207 @@
+        callbackSuccess = QUOTE([ARR_3(_player, _patient, 'Lorazepam')] call FUNC(treatmentAdvanced_Generic));
+    };
+    class Flumazenil: Carbonate {
+        displayName = "Push Flumazenil";
```
CSTRING

MiszczuZPolski commented on addons/surgery/ACE_Medical_Treatment_Actions.hpp:
```diff
@@ -0,0 +1,207 @@
+        callbackSuccess = QUOTE([ARR_3(_player, _patient, 'Flumazenil')] call FUNC(treatmentAdvanced_Generic));
+    };
+    class Expose: BasicBandage {
+        displayName = "Expose fracture";
```
CSTRING

MiszczuZPolski commented on addons/surgery/ACE_Medical_Treatment_Actions.hpp:
```diff
@@ -0,0 +1,207 @@
+    };
+    class Expose: BasicBandage {
+        displayName = "Expose fracture";
+        displayNameProgress = "Exposing";
```
CSTRING

MiszczuZPolski commented on addons/surgery/ACE_Medical_Treatment_Actions.hpp:
```diff
@@ -0,0 +1,207 @@
+        callbackSuccess = QUOTE([ARR_4(_medic, _patient, _bodyPart, 2.1)] call FUNC(openFractureProgress));
+    };
+    class Incision: BasicBandage {
+        displayName = "Peform Incision";
```
CSTRING

MiszczuZPolski commented on addons/surgery/ACE_Medical_Treatment_Actions.hpp:
```diff
@@ -0,0 +1,207 @@
+    };
+    class Incision: BasicBandage {
+        displayName = "Peform Incision";
+        displayNameProgress = "Performing Incision";
```
CSTRING

MiszczuZPolski commented on addons/surgery/ACE_Medical_Treatment_Actions.hpp:
```diff
@@ -0,0 +1,207 @@
+        callbackSuccess = QUOTE([ARR_3(_medic, _patient, _bodyPart)] call FUNC(openFractureIncision));
+    };
+    class Clamp: BasicBandage {
+        displayName = "Clamp Fracture";
```
CSTRING

MiszczuZPolski commented on addons/surgery/ACE_Medical_Treatment_Actions.hpp:
```diff
@@ -0,0 +1,207 @@
+    };
+    class Clamp: BasicBandage {
+        displayName = "Clamp Fracture";
+        displayNameProgress = "Clamping Fracture";
```
CSTRING

MiszczuZPolski commented on addons/surgery/ACE_Medical_Treatment_Actions.hpp:
```diff
@@ -0,0 +1,207 @@
+        callbackSuccess = QUOTE([ARR_4(_medic, _patient, _bodyPart, 3.3)] call FUNC(openFractureProgress));
+    };
+    class Irrigate: BasicBandage {
+        displayName = "Irrigate Wound";
```
CSTRING

MiszczuZPolski commented on addons/surgery/ACE_Medical_Treatment_Actions.hpp:
```diff
@@ -0,0 +1,207 @@
+    };
+    class Irrigate: BasicBandage {
+        displayName = "Irrigate Wound";
+        displayNameProgress = "Irrigating";
```
CSTRING

MiszczuZPolski commented on addons/surgery/ACE_Medical_Treatment_Actions.hpp:
```diff
@@ -0,0 +1,207 @@
+        callbackSuccess = QUOTE([ARR_4(_medic, _patient, _bodyPart, 2.3)] call FUNC(openFractureProgress));
+    };
+    class Debridement: BasicBandage {
+        displayName = "Debride Wounds";
```
CSTRING

MiszczuZPolski commented on addons/surgery/ACE_Medical_Treatment_Actions.hpp:
```diff
@@ -0,0 +1,207 @@
+    };
+    class Debridement: BasicBandage {
+        displayName = "Debride Wounds";
+        displayNameProgress = "Debriding";
```
CSTRING

MiszczuZPolski commented on addons/surgery/ACE_Medical_Treatment_Actions.hpp:
```diff
@@ -0,0 +1,207 @@
+        allowedSelections[] = {"Head", "Body", "LeftArm", "RightArm", "LeftLeg", "RightLeg"};
+        allowSelfTreatment = 0;
+        medicRequired = QGVAR(surgicalAction);
+        treatmentTime = 15;
```
make it settable

MiszczuZPolski commented on addons/surgery/ACE_Medical_Treatment_Actions.hpp:
```diff
@@ -0,0 +1,207 @@
+        callbackSuccess = QUOTE([ARR_3(_medic, _patient, _bodyPart)] call FUNC(debridementClear));
+    };
+    class NPWT: BasicBandage {
+        displayName = "Apply NPWT Dressing";
```
CSTRING

MiszczuZPolski commented on addons/surgery/ACE_Medical_Treatment_Actions.hpp:
```diff
@@ -0,0 +1,207 @@
+    };
+    class NPWT: BasicBandage {
+        displayName = "Apply NPWT Dressing";
+        displayNameProgress = "Applying";
```
CSTRING

MiszczuZPolski commented on addons/surgery/ACE_Medical_Treatment_Actions.hpp:
```diff
@@ -0,0 +1,207 @@
+        category = "surgery";
+        treatmentLocations = QGVAR(surgicalLocation);
+        allowedSelections[] = {"Head", "Body", "LeftArm", "RightArm", "LeftLeg", "RightLeg"};
+        allowSelfTreatment = 8;
```
8? it should be 0/1

MiszczuZPolski commented on addons/surgery/ACE_Medical_Treatment_Actions.hpp:
```diff
@@ -0,0 +1,207 @@
+        allowedSelections[] = {"Head", "Body", "LeftArm", "RightArm", "LeftLeg", "RightLeg"};
+        allowSelfTreatment = 8;
+        medicRequired = QGVAR(surgicalAction);
+        treatmentTime = 8;
```
make it settable

MiszczuZPolski commented on addons/surgery/CfgWeapons.hpp:
```diff
@@ -0,0 +1,93 @@
+
+    class kat_etomidate: ACE_ItemCore {
+        scope = 2;
+        displayName = "Etomidate";
```
CSTRING

MiszczuZPolski commented on addons/surgery/CfgWeapons.hpp:
```diff
@@ -0,0 +1,93 @@
+    };
+    class kat_lorazepam: ACE_ItemCore {
+        scope = 2;
+        displayName = "Lorazepam";
```
CSTRING

MiszczuZPolski commented on addons/surgery/CfgWeapons.hpp:
```diff
@@ -0,0 +1,93 @@
+    };
+    class kat_flumazenil: ACE_ItemCore {
+        scope = 2;
+        displayName = "Flumazenil";
```
CSTRING

MiszczuZPolski commented on addons/surgery/CfgWeapons.hpp:
```diff
@@ -0,0 +1,93 @@
+    };
+    class kat_scalpel: ACE_ItemCore {
+        scope = 2;
+        displayName = "Scalpel";
```
CSTRING

MiszczuZPolski commented on addons/surgery/CfgWeapons.hpp:
```diff
@@ -0,0 +1,93 @@
+    };
+    class kat_retractor: ACE_ItemCore {
+        scope = 2;
+        displayName = "Retractor";
```
CSTRING

MiszczuZPolski commented on addons/surgery/CfgWeapons.hpp:
```diff
@@ -0,0 +1,93 @@
+    };
+    class kat_clamp: ACE_ItemCore {
+        scope = 2;
+        displayName = "Clamp";
```
CSTRING

MiszczuZPolski commented on addons/surgery/CfgWeapons.hpp:
```diff
@@ -0,0 +1,93 @@
+    };
+    class kat_plate: ACE_ItemCore {
+        scope = 2;
+        displayName = "Bone Plate";
```
CSTRING

MiszczuZPolski commented on addons/surgery/CfgWeapons.hpp:
```diff
@@ -0,0 +1,93 @@
+    };
+    class kat_vacuum: ACE_ItemCore {
+        scope = 2;
+        displayName = "Dressing Vacuum";
```
CSTRING

MiszczuZPolski commented on addons/surgery/XEH_preInit.sqf:
```diff
@@ -0,0 +1,146 @@
+#include "XEH_PREP.hpp"
+PREP_RECOMPILE_END;
+
+#define CBA_SETTINGS_CAT_M "KAT Medical - Surgery"
```
"KAT - ADV Medical: Surgery"

MiszczuZPolski commented on addons/surgery/functions/fnc_fractureSelectLocal.sqf:
```diff
@@ -0,0 +1,66 @@
+    private _subValue = random 100;
+    if (_subValue <= GVAR(compoundChance)) then {
+        _liveFracture = 2;
+        _fractureString = "Compound Fracture";
```
CSTRING?

MiszczuZPolski commented on addons/surgery/functions/fnc_fractureSelectLocal.sqf:
```diff
@@ -0,0 +1,66 @@
+        _fractureString = "Compound Fracture";
+    } else {
+        _liveFracture = 3;
+        _fractureString = "Comminuted Fracture";
```
CSTRING?

MiszczuZPolski commented on addons/surgery/functions/fnc_fractureSelectLocal.sqf:
```diff
@@ -0,0 +1,66 @@
+
+if (_value <= GVAR(simpleChance)) then {
+    _liveFracture = 1;
+    _fractureString = "Simple Fracture";
```
CSTRING

MiszczuZPolski commented on addons/surgery/functions/fnc_fractureSelectLocal.sqf:
```diff
@@ -0,0 +1,66 @@
+    };
+
+    if (_liveFracture >= 2 && _liveFracture < 3) then {
+        _fractureString = "Compound Fracture";
```
CSTRING?

MiszczuZPolski commented on addons/surgery/functions/fnc_fractureSelectLocal.sqf:
```diff
@@ -0,0 +1,66 @@
+    };
+
+    if (_liveFracture == 1) then {
+        _fractureString = "Simple Fracture";
```
CSTRING?

MiszczuZPolski commented on addons/surgery/functions/fnc_fractureSelectLocal.sqf:
```diff
@@ -0,0 +1,66 @@
+if (_liveFracture != 0) exitWith {
+
+    if (_liveFracture >= 3) then  {
+        _fractureString = "Comminuted Fracture";
```
CSTRING?

MiszczuZPolski commented on addons/surgery/functions/fnc_openFractureProgressLocal.sqf:
```diff
@@ -0,0 +1,79 @@
+};
+
+if (_entry == 2.3 && _reduce) exitWith {
+    private _output = localize LSTRING(fracture_fail);
```
LLSTRING

MiszczuZPolski commented on addons/surgery/functions/fnc_openFractureProgressLocal.sqf:
```diff
@@ -0,0 +1,79 @@
+if (_number == _liveFracture) exitWith {
+    switch (true) do {
+        case (_entry == 2.1): {
+        _surgeryString = "exposed";
```
CSTRING

MiszczuZPolski commented on addons/surgery/functions/fnc_openFractureProgressLocal.sqf:
```diff
@@ -0,0 +1,79 @@
+        _surgeryString = "exposed";
+        };
+        case (_entry == 2.3): {
+        _surgeryString = "irrigated";
```
CSTRING

MiszczuZPolski commented on addons/surgery/functions/fnc_openFractureProgressLocal.sqf:
```diff
@@ -0,0 +1,79 @@
+        _surgeryString = "irrigated";
+        };
+        case (_entry == 3.3): {
+        _surgeryString = "clamped";
```
CSTRING

MiszczuZPolski commented on addons/surgery/script_component.hpp:
```diff
@@ -0,0 +1,36 @@
+#define COMPONENT circulation
+#define COMPONENT_BEAUTIFIED ACE - Circulation
+#include "\x\aceP\addons\main\script_mod.hpp"
```
aceP?

MiszczuZPolski commented on addons/surgery/script_component.hpp:
```diff
@@ -0,0 +1,36 @@
+    #define DEBUG_SETTINGS DEBUG_SETTINGS_CIRCULATION
+#endif
+
+#include "\x\aceP\addons\main\script_macros.hpp"
```
aceP?

MiszczuZPolski commented on addons/surgery/ACE_Medical_Treatment_Actions.hpp:
```diff
@@ -0,0 +1,207 @@
+        allowedSelections[] = {"Head"};
+        allowSelfTreatment = 1;
+        medicRequired = 1;
+        treatmentTime = 2;
```
make it settable

MiszczuZPolski commented on addons/surgery/ACE_Medical_Treatment_Actions.hpp:
```diff
@@ -0,0 +1,207 @@
+        displayNameProgress = "Performing";
+        items[] = {};
+        condition = QFUNC(vehicleCheckDialysis);
+        treatmentTime = 15;
```
make it settable

MiszczuZPolski commented on addons/surgery/ACE_Medical_Treatment_Actions.hpp:
```diff
@@ -0,0 +1,207 @@
+        items[] = {};
+        condition = QFUNC(vehicleCheckDialysis);
+        treatmentTime = 15;
+        medicRequired = 1;
```
make it settable

MiszczuZPolski commented on addons/surgery/ACE_Medical_Treatment_Actions.hpp:
```diff
@@ -0,0 +1,207 @@
+        allowedSelections[] = {"LeftArm", "RightArm", "LeftLeg", "RightLeg"};
+        allowSelfTreatment = 1;
+        medicRequired = QGVAR(fractureCheck_Level);
+        treatmentTime = 5;
```
make it settable

MiszczuZPolski commented on addons/surgery/ACE_Medical_Treatment_Actions.hpp:
```diff
@@ -0,0 +1,207 @@
+        treatmentLocations = 0;
+        allowedSelections[] = {"Body", "LeftArm", "RightArm", "LeftLeg", "RightLeg"};
+        allowSelfTreatment = 0;
+        medicRequired = 1;
```
make it settable

MiszczuZPolski commented on addons/surgery/ACE_Medical_Treatment_Actions.hpp:
```diff
@@ -0,0 +1,207 @@
+        treatmentLocations = 0;
+        allowedSelections[] = {"Body", "LeftArm", "RightArm", "LeftLeg", "RightLeg"};
+        allowSelfTreatment = 0;
+        medicRequired = 1;
```
make it settable

MiszczuZPolski commented on addons/surgery/ACE_Medical_Treatment_Actions.hpp:
```diff
@@ -0,0 +1,207 @@
+        treatmentLocations = 0;
+        allowedSelections[] = {"Body", "LeftArm", "RightArm", "LeftLeg", "RightLeg"};
+        allowSelfTreatment = 0;
+        medicRequired = 1;
```
make it settable

MiszczuZPolski reviewed (changes requested):
Quick review
Cope, bird boy

mazinskihenry committed:
Header update and cleaning

mazinskihenry committed:
Frog man smells like swamp

YetheSamartaka commented:
100% agree with Miszczu.

mazinskihenry committed:
Merge branch 'Tomcat-SG:dev-Tomcat' into surgery

mazinskihenry committed:
Merge branch 'Tomcat-SG:dev-Tomcat' into surgery

YetheSamartaka committed:
handleRespawn for Surgery

- fills missing variables for handleRespawn function

YetheSamartaka committed:
fullHealLocal for Surgery 

- fills missing variables for fullHealLocal function

MiszczuZPolski commented on addons/surgery/functions/fnc_openFractureProgressLocal.sqf:
```diff
@@ -0,0 +1,77 @@
+    _patient setVariable [QGVAR(fractures), _fractureArray, true];
+};
+
+private _output = localize LSTRING(fracture_fail);
```
LLSTRING

MiszczuZPolski commented on addons/surgery/functions/fnc_openFractureProgressLocal.sqf:
```diff
@@ -0,0 +1,77 @@
+params ["_medic", "_patient", "_bodyPart", "_entry"];
+
+if !(IS_UNCONSCIOUS(_patient)) exitWith {
+    private _output = localize LSTRING(fracture_fail);
```
LLSTRING

YetheSamartaka committed:
fixed performing to perform as in stringtable

YetheSamartaka committed:
small fixes

YetheSamartaka committed:
stringtable fixes

YetheSamartaka committed:
new addon options + small fix

YetheSamartaka committed:
new addon options + small adjustments

- added new addon options
- slightly adjusted one category
- changed default medical level required for all interactions to the lowest level

YetheSamartaka committed:
new stringtables for new addon options

YetheSamartaka committed:
LSTRING and LLSTRING utilisation

YetheSamartaka committed:
LSTRING utilisation

YetheSamartaka committed:
CSTRING utilisation

YetheSamartaka committed:
new stringtables for cfgWeapons

YetheSamartaka committed:
missing ,

YetheSamartaka committed:
fixed wrong description of fracture check level to timer

YetheSamartaka committed:
surgicalLocation changed to Anywhere in default settings

mazinskihenry committed:
More stuff

mazinskihenry committed:
hmm

YetheSamartaka committed:
fix condition to allow Arma to even launch without error

YetheSamartaka committed:
Changed default settings for self check fracture to true

YetheSamartaka committed:
Merge branch 'dev-Tomcat' into surgery

YetheSamartaka committed:
reverting this mess

YetheSamartaka committed:
Merge branch 'surgery' of https://github.com/mazinskihenry/KAM into pr/122

YetheSamartaka committed:
Reapplying relevant changes

:void:

YetheSamartaka committed:
missing Dialysis functions

just forgot to include those files

YetheSamartaka committed:
treatment actions tweaks

- Fixed condition for lorazepam, Flumanezil and Etomidate
- Expanded conditions to take into account the requiere IV for certain medications
- Allowed surgery actions to be visible always(To prevent cheating by just seeing the "correct" actions)
- Wrong naming for medicRequiered + one missing
- Added setting for requiering unconsciousness to perform surgery actions

Tomcat-SG committed:
space

Tomcat-SG committed:
space

Tomcat-SG commented on addons/pharma/ACE_Medical_Treatment_Actions.hpp:
```diff
@@ -50,6 +50,17 @@ class ACE_Medical_Treatment_Actions {
         callbackSuccess = QUOTE([ARR_3(_player, _patient, 'Naloxone')] call FUNC(treatmentAdvanced_Naloxone));
     };
+    class EACA: Carbonate {
+        displayName = CSTRING(Take_EACA);
```
string is missing in stringtable

Tomcat-SG commented on addons/pharma/ACE_Medical_Treatment_Actions.hpp:
```diff
@@ -59,7 +70,46 @@ class ACE_Medical_Treatment_Actions {
+        callbackSuccess = QFUNC(treatmentAdvanced_TXA);
+    };
+class Saline_Flush: Carbonate {
+        displayName = "Saline Flush";
```
CSTRING

Tomcat-SG commented on addons/pharma/ACE_Medical_Treatment_Actions.hpp:
```diff
@@ -59,7 +70,46 @@ class ACE_Medical_Treatment_Actions {
+        callbackSuccess = QFUNC(treatmentAdvanced_Flush);
+    };
+    class Inspect: Carbonate {
+        displayName = "Inspect Catheter";
```
CSTRING

Tomcat-SG commented on addons/pharma/ACE_Medical_Treatment_Actions.hpp:
```diff
@@ -59,7 +70,46 @@ class ACE_Medical_Treatment_Actions {
+        animationMedicProne = "";
+    };
+    class BreathCheck: Carbonate {
+        displayName = "Check Breath";
```
CSTRING

Tomcat-SG commented on addons/pharma/CfgWeapons.hpp:
```diff
@@ -51,6 +51,17 @@ class CfgWeapons {
     };
+    class kat_EACA: ACE_ItemCore {
+        scope = 2;
+        displayName = "EACA";
```
CSTRING

Tomcat-SG commented on addons/pharma/CfgWeapons.hpp:
```diff
@@ -51,6 +51,17 @@ class CfgWeapons {
+        displayName = "EACA";
+        picture = QPATHTOF(ui\icon_TXA.paa);
+        model = "\A3\Structures_F_EPA\Items\Medical\Painkillers_F.p3d";
+        descriptionShort = CSTRING(EACA_DescShort);
```
string is missing in stringtable

Tomcat-SG commented on addons/pharma/CfgWeapons.hpp:
```diff
@@ -51,6 +51,17 @@ class CfgWeapons {
+        picture = QPATHTOF(ui\icon_TXA.paa);
+        model = "\A3\Structures_F_EPA\Items\Medical\Painkillers_F.p3d";
+        descriptionShort = CSTRING(EACA_DescShort);
+        descriptionUse = CSTRING(EACA_DescUse);
```
string is missing in stringtable

Tomcat-SG committed:
space

Tomcat-SG commented on addons/pharma/XEH_preInit.sqf:
```diff
@@ -138,5 +138,40 @@ PREP_RECOMPILE_END;
+[
+	QGVAR(aiEnableAdvanced),
+	"CHECKBOX",
+	["Enable Advanced Medical for AI", "Enables kidney actions and blood pressure PFHs for AI"],
```
LLSTRING

Tomcat-SG commented on addons/pharma/functions/fnc_alphaAction.sqf:
```diff
@@ -0,0 +1,34 @@
+ * Total blood loss of unit (litres/second) <NUMBER>
+ *
+ * Example:
+ * [player] call ace_medical_status_fnc_getBloodLoss
```
not correct

Tomcat-SG commented on addons/pharma/functions/fnc_alphaAction.sqf:
```diff
@@ -0,0 +1,34 @@
+ * 0: The Unit <OBJECT>
+ *
+ * Return Value:
+ * Total blood loss of unit (litres/second) <NUMBER>
```
not correct

Tomcat-SG commented on addons/pharma/functions/fnc_alphaAction.sqf:
```diff
@@ -0,0 +1,34 @@
+ * Calculate the total blood loss of a unit.
+ *
+ * Arguments:
+ * 0: The Unit <OBJECT>
```
not correct

Tomcat-SG commented on addons/pharma/functions/fnc_alphaAction.sqf:
```diff
@@ -0,0 +1,34 @@
+#include "script_component.hpp"
+/*
+ * Author: Glowbal
+ * Calculate the total blood loss of a unit.
```
not correct

YetheSamartaka replied:
@MiszczuZPolski will resolve it in his Pharmacy refactor.

Tomcat-SG commented on addons/pharma/functions/fnc_inspectCatheter.sqf:
```diff
@@ -0,0 +1,33 @@
+ * Author: 2LT.Mazinski
+ * Opens an IV/IO on a patient and changes the patient's flow variable
+ *
+ * Arguments:
```
arguments not correct

Tomcat-SG commented on addons/pharma/functions/fnc_inspectCatheter.sqf:
```diff
@@ -0,0 +1,33 @@
+#include "script_component.hpp"
+/*
+ * Author: 2LT.Mazinski
+ * Opens an IV/IO on a patient and changes the patient's flow variable
```
description not correct

Tomcat-SG commented on addons/pharma/functions/fnc_inspectCatheter.sqf:
```diff
@@ -0,0 +1,33 @@
+ * None
+ *
+ * Example:
+ * [player, cursorObject, "LeftLeg", "", objNull, "kat_IV_20"] call kat_circulation_fnc_fractureCheck;
```
example not correct

Tomcat-SG commented on addons/pharma/functions/fnc_salineCheck.sqf:
```diff
@@ -0,0 +1,35 @@
+#include "script_component.hpp"
+/*
+ * Author: 2LT.Mazinski
+ * Opens an IV/IO on a patient and changes the patient's flow variable
```
description not correct

Tomcat-SG commented on addons/pharma/functions/fnc_salineCheck.sqf:
```diff
@@ -0,0 +1,35 @@
+ * Author: 2LT.Mazinski
+ * Opens an IV/IO on a patient and changes the patient's flow variable
+ *
+ * Arguments:
```
arguments not correct

Tomcat-SG commented on addons/pharma/functions/fnc_salineCheck.sqf:
```diff
@@ -0,0 +1,35 @@
+ * 2: Body Part <STRING>
+ *
+ * Return Value:
+ * None
```
Return Value not correct

Tomcat-SG commented on addons/pharma/functions/fnc_salineCheck.sqf:
```diff
@@ -0,0 +1,35 @@
+ * None
+ *
+ * Example:
+ * [player, cursorObject, "LeftLeg"] call kat_circulation_fnc_applyIV;
```
Example not correct

Tomcat-SG commented on addons/pharma/functions/fnc_treatmentAdvanced_EACA.sqf:
```diff
@@ -0,0 +1,27 @@
+#include "script_component.hpp"
+/*
+ * Author: Katalam
+ * Airway Management for occluding
```
description not correct

Tomcat-SG commented on addons/pharma/functions/fnc_treatmentAdvanced_EACA.sqf:
```diff
@@ -0,0 +1,27 @@
+ * Succesful treatment started <BOOL>
+ *
+ * Example:
+ * [player, cursorTarget] call kat_airway_fnc_treatmentAdvanced_accuvac;
```
example not correct

Tomcat-SG commented on addons/pharma/functions/fnc_treatmentAdvanced_EACA.sqf:
```diff
@@ -0,0 +1,27 @@
+params ["_medic", "_patient"];
+
+if (local _patient) then {
+    ["treatmentEACA", [_patient, "EACA", _medic]] call CBA_fnc_localEvent;
```
eventhandler is missing in the XEH_postInit!

Tomcat-SG commented on addons/pharma/functions/fnc_treatmentAdvanced_EACALocal.sqf:
```diff
@@ -0,0 +1,80 @@
+#include "script_component.hpp"
+/*
+ * Author: 2LT.Mazinski
+ * Begins TXA bandaging process
```
description not correct

Tomcat-SG commented on addons/pharma/functions/fnc_treatmentAdvanced_EACALocal.sqf:
```diff
@@ -0,0 +1,80 @@
+ * Begins TXA bandaging process
+ *
+ * Arguments:
+ * 0: Medic <OBJECT>
```
arguments not correct

Tomcat-SG commented on addons/pharma/functions/fnc_treatmentAdvanced_EACALocal.sqf:
```diff
@@ -0,0 +1,80 @@
+ * 1: Medication <STRING>
+ *
+ * Return Value:
+ * None
```
Return Value not correct

Tomcat-SG commented on addons/pharma/functions/fnc_treatmentAdvanced_EACALocal.sqf:
```diff
@@ -0,0 +1,80 @@
+ * None
+ *
+ * Example:
+ * [player, "TXA"] call aceP_circulation_fnc_treatmentAdvanced_TXALocal;
```
Example not correct

Tomcat-SG commented on addons/pharma/functions/fnc_vehicleCheckDialysis.sqf:
```diff
@@ -0,0 +1,30 @@
+ * 1: Patient <OBJECT>
+ *
+ * Return Value:
+ * None
```
Return Value not correct

Tomcat-SG committed:
space

Tomcat-SG commented on addons/surgery/XEH_postInit.sqf:
```diff
@@ -0,0 +1,14 @@
+#include "script_component.hpp"
+
+["treatmentKetamine", {_this call FUNC(treatmentAdvanced_KetamineLocal)}] call CBA_fnc_addEventHandler;
+["treatmentLido", {_this call FUNC(treatmentAdvanced_LidoLocal)}] call CBA_fnc_addEventHandler;
```
where do they get called?

Tomcat-SG commented on addons/surgery/functions/fnc_closedFractureLocal.sqf:
```diff
@@ -0,0 +1,48 @@
+ * Author: 1LT.Mazinski.H
+ * Local call for fully healing a fracture.
+ *
+ * Arguments:
```
param entry is missing

Tomcat-SG committed:
space

Tomcat-SG committed:
plate was missing

Tomcat-SG commented on addons/surgery/functions/fnc_debridementClearLocal.sqf:
```diff
@@ -0,0 +1,44 @@
+ * None
+ *
+ * Example:
+ * [player, cursorObject, "LeftLeg"] call kat_surgery_fnc_debridementCheckLocal;
```
```suggestion
 * [player, cursorObject, "LeftLeg"] call kat_surgery_fnc_debridementClearLocal;
```

Tomcat-SG commented on addons/surgery/functions/fnc_openFractureProgress.sqf:
```diff
@@ -0,0 +1,26 @@
+ * Nothing
+ *
+ * Example:
+ * [player, cursorObject, "LeftLeg"] call kat_surgery_fnc_openFractureProgress
```
is param 3 optional?

Tomcat-SG commented on addons/surgery/functions/fnc_init.sqf:
```diff
@@ -0,0 +1,35 @@
+ * None
+ *
+ * Example:
+ * [player] call kat_circulation_fnc_init;
```
not correct example

Tomcat-SG commented on addons/surgery/functions/fnc_fractureCheck.sqf:
```diff
@@ -0,0 +1,33 @@
+ * Arguments:
+ * 0: Medic <OBJECT>
+ * 1: Patient <OBJECT>
+ * 2: Body Part <STRING>
```
param 3 is missing

Tomcat-SG commented on addons/surgery/functions/fnc_openFractureCheck.sqf:
```diff
@@ -0,0 +1,34 @@
+ * 3: Fracture Status <NUMBER>
+ *
+ * Return Value:
+ * None
```
missing return value

Tomcat-SG reviewed (changes requested):
Need to update the missing and wrong things. Pls also check the DEV Discord Channel @Devs

YetheSamartaka committed:
stringtables for EACA

YetheSamartaka committed:
Added missing stringtables

YetheSamartaka committed:
LLSTRING for AI kidney actions

YetheSamartaka committed:
Added missing stringtables

YetheSamartaka committed:
Update fnc_inspectCatheter.sqf

YetheSamartaka committed:
Update fnc_salineCheck.sqf

YetheSamartaka committed:
Update fnc_treatmentAdvanced_EACA.sqf

YetheSamartaka committed:
Update fnc_treatmentAdvanced_EACALocal.sqf

YetheSamartaka committed:
Update fnc_vehicleCheckDialysis.sqf

YetheSamartaka committed:
Update fnc_closedFractureLocal.sqf

YetheSamartaka committed:
Update addons/surgery/functions/fnc_debridementClearLocal.sqf

Co-authored-by: Tomcat <51529018+Tomcat-SG@users.noreply.github.com>

YetheSamartaka committed:
Update fnc_openFractureProgress.sqf

YetheSamartaka committed:
Update fnc_init.sqf

YetheSamartaka committed:
Update fnc_fractureCheck.sqf

YetheSamartaka committed:
Update fnc_openFractureCheck.sqf

YetheSamartaka committed:
Deletion of unused things
